zipkin-autoconfigure/collector-rabbitmq/src/test/java/zipkin/autoconfigure/collector/rabbitmq/ZipkinRabbitMQCollectorAutoConfigurationTest.java

```diff
@@ -58,9 +58,9 @@ public void doesNotProvideCollectorComponent_whenAddressNotSet() {
   }
 
```

Comment:
maybe just for good measure, add a test that when you set URI it is readable and hosts is ignored?

Reply:
Added https://github.com/openzipkin/zipkin/pull/1922/files#diff-e3e00988ebb1a8c1c756f3a67a6d533f